translator/opencensus/oc_to_internal.go

```diff
@@ -0,0 +1,405 @@
+// Copyright 2019 OpenTelemetry Authors
```

Comment:
nit: 2020

Reply:
There is an ongoing discussion elsewhere about getting rid of the year. Let's wait until the resolution.

Reply:
done